rocAL/rocAL_pybind/CMakeLists.txt

```diff
@@ -19,7 +19,10 @@
 # THE SOFTWARE.
 
 cmake_minimum_required(VERSION 3.0)
-project(rocal_pybind)
+project(rocAL_pybind)
+
+set(TARGET_NAME rocAL_pybind)
```

Comment:
also here

Reply:
I will have to change the name of the folder also to reflect 'rocal_pybind'. That ok?